app/src/main/java/com/springboot/webpush/service/NotificationService.java

```diff
@@ -19,24 +19,41 @@
 import nl.martijndwars.webpush.Notification;
 import nl.martijndwars.webpush.PushService;
 import nl.martijndwars.webpush.Urgency;
+import java.security.KeyPair;
+import java.security.PublicKey;
 
 @Service
 public class NotificationService {
 	private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
 	private final SubscriptionService subscriptionService;
 	private final ObjectMapper objectMapper;
-	private final PushService pushService;
+	private PushService pushService;
+	private final KeyPair keyPair;
 
 	public NotificationService(SubscriptionService subscriptionService, ObjectMapper objectMapper,
 			final KeyPairService keyPairService) {
 		super();
 		this.subscriptionService = subscriptionService;
 		this.objectMapper = objectMapper;
+		this.keyPair = keyPairService.generate().getKeyPair();
+
+		/*
+		 * this.pushService = new PushService(this.keyPair) //
+		 * .setSubject("mailto:admin@domain.com");
+		 */
+		try {
+			this.pushService = new PushService(
+					"BJ3kKgvZR34Wt-dJMG9L7Lh03ISRlWueIgg-Vp5V5oYsDPr47_szfNsxxLQrghPtuN3XaI6A3FVZVZuVa7LZOFY",
+					"pMicnxO7KOCxUjbA6--6S1R5NCbnyE4A0-EZxravdGA").setSubject("mailto:admin@domain.com");
+		} catch (GeneralSecurityException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
 
-		this.pushService = new PushService(keyPairService.generate().getKeyPair()) //
-				.setSubject("mailto:admin@domain.com");
-
+	public PublicKey getPublicKey() {
+		return this.keyPair.getPublic();
 	}
 
 	public void notifyAll(WebPushMessage message) {
```

Comment:
This WebPushMessage is totally wrong (or maybe just outdated). Should end up looking like:
```
{
  notification: Notification
}
```
where Notification is defined here: https://developer.mozilla.org/en-US/docs/Web/API/Notification